ldclient.go

```diff
@@ -34,6 +34,9 @@ import (
 // Version is the SDK version.
 const Version = internal.SDKVersion
 
+// HighWaitForSeconds is the initialization wait time threshold that we will log a warning message
+const HighWaitForSeconds = 60
```

Comment:
```suggestion
// highWaitForSeconds is the initialization wait time threshold that we will log a warning message
const highWaitForSeconds = 60
```
Uppercase = public variable. Totally intuitive, right?

Anyways, I think we want this private unless there is a good reason to expose it.